src/Components/test/testassets/Components.TestServer/RazorComponentEndpointsStartup.cs

```diff
@@ -59,8 +58,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseEndpoints(endpoints =>
             {
                 endpoints.MapRazorComponents<TRootComponent>()
+                    .AddAdditionalAssemblies(Assembly.Load("Components.WasmMinimal"))
                     .AddServerRenderMode()
-                    .AddWebAssemblyRenderMode();
+                    .AddWebAssemblyRenderMode(new WebAssemblyComponentsEndpointOptions
+                    {
+                        PathPrefix = "/WasmMinimal"
```

Comment:
`AddWebAssemblyComponents` is the much more natural place to configure options since it follows the options pattern and aligns nicely with `AddServerComponents`.

If we're going to do this last-minute non-API-approved change, shouldn't we at least test what happens if you need per-`MapRazorComponents` `WebAssmblyComponentsEndpointOptions.PathPrefix` configuration? And verify everything works if you call `MapRazorComponents` multiple times with different path prefixes. This is the kind of thing that often doesn't work if you don't test it. I had to do extra work to make `MapIdentityApi` work if you called it with multiple different `TUser` types that I would have never found without testing.

If this isn't something that is so difficult and unusual that we cannot even test it, we're better off going with the global option we had before. If we need per-`MapRazorComponents` of this in the future, we could always add a global override that works exactly like this.

Reply:
I just tested this locally and it works. IMO, making this an option on the endpoint is ideal because hosting multiple WebAssembly apps is something we [already support and document](https://learn.microsoft.com/aspnet/core/blazor/host-and-deploy/multiple-hosted-webassembly), and ideally we'd want an equivalent pattern with these new APIs.

I can see that there's still some deviation from the original API proposal with this PR, so I'm going to update the API review issue to match these changes and we can make a final decision on Monday.

Reply:
I'm glad it works. I'm fine with keeping it. We should probably make `AddServerRenderMode` an extension method as well for consistency. But we can merge this as-is for now and make any final adjustments after we API review it.

Reply:
And we add an automated test for this too then?

Reply:
> And can we add an automated test for this too then?

For sure. Is it fine if we do that in a follow-up? It's actually a bit involved, and I probably won't be able to finish by EOD.

Reply:
That's fine with me.